config/jobs/README.md

```diff
@@ -88,12 +82,13 @@ presubmits:
     always_run: true
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/gcloud-in-go:v20190125-cc5d6ecff3
+      - image: golang:1.12.5
         command:
         - /bin/bash
         args:
         - -c
-        - "make verify"
+        # Add GOPATH/bin back to PATH to workaround #9469
+        - "export PATH=$GOPATH/bin:$PATH && make verify"
```

Comment:
Updated based on https://github.com/kubernetes/test-infra/pull/13134

Reply:
I thought the `go` images had path set up correctly?

Reply:
When we decorate the pod we set `GOPATH` to `/home/prow/go`, so even though`/go/bin` is in this image's `PATH`, that's not where `go install` will put things.  I reopened #9469

Reply:
I thought we used /go,  huh